### PR TITLE
Added : landing page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (67)
Screenshot (68)
Screenshot (69)

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
approved

File: landing page/index.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    margin-right: 5rem;
+    padding: 0rem 1rem;
+    background-color:var(--primary-color);
+    min-width: fit-content;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+  }
+  .dropdown-content a{
+    float: none;
+    padding: 12px 16px;
```
Comment (ShrutiLakhani) [inline landing page/index.css]: Please give it in rem for anything over 10px

File: landing page/index.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    position: absolute;
+    margin-right: 5rem;
+    padding: 0rem 1rem;
+    background-color:var(--primary-color);
+    min-width: fit-content;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+  }
+  .dropdown-content a{
+    float: none;
```
Comment (ShrutiLakhani) [inline landing page/index.css]: We should avoid using float. Please check

**Review by ShrutiLakhani — APPROVED** [review]
Code looks clean to merge. Please check on few suggestions made

**Pull request merged**